frontend/device/sdl/device.lua

```diff
@@ -120,6 +120,12 @@ local Desktop = Device:extend{
     hasExitOptions = notOSX,
 }
 
+local Flatpak = Device:extend{
+    model = "Flatpak",
+    isDesktop = yes,
+    canExternalDictLookup = no,
```

Comment:
What would be the Flatpak way of doing that btw, if you know?

Reply:
`--filesystem=host-os` let you access `/usr` binaries linked somewhere within the app sandbox. They're not on the PATH so it requires another method of probing dicts.

Also, the kind of people that uses KOReader from a flatpak might have goldendict sandboxed too, so it won't work at all.

I'm mostly happy disabling 3rd party dicts. `xdg-open` works fine to open links in the browser, so hey :)